test/jdk/java/net/URLPermission/nstest/LookupTest.java

```diff
@@ -100,6 +104,23 @@ static void test(String url,
         }
     }
 
+    private static void configureSystemWideNoProxy() {
+        ProxySelector.setDefault(new ProxySelector() {
```

Comment:
Hello Volkan, `ProxySelector.setDefault(ProxySelector.of(null))` allows you to disable proxy without requiring the code to provide a `ProxySelector` implementation. Maybe that could be used here? That might also mean you won't need a separate `configureSystemWideNoProxy()` method.